src/zm_image.cpp

```diff
@@ -2001,7 +2001,7 @@ void Image::Annotate(
   const uint8_t bg_b_col = BLUE_VAL_RGBA(bg_colour);
   const uint8_t bg_bw_col = bg_colour & 0xff;
   const Rgb bg_rgb_col = rgb_convert(bg_colour, subpixelorder);
-  const bool bg_trans = (bg_colour == RGB_TRANSPARENT);
+  const bool bg_trans = (bg_colour == kRGBTransparent);
```

Comment:
this stuff seems unrelated to fmt.  What's the k signify?

Reply:
I rebased the PR, so those changes are gone now since https://github.com/ZoneMinder/zoneminder/pull/3181 is merged.
The "k" prefix is used to distinguish constants (see Google styleguide: https://google.github.io/styleguide/cppguide.html#Constant_Names)